R/mod_edish.R

```diff
@@ -492,6 +492,17 @@ check_mod_edish <- function(
     lb_result_var, ref_range_upper_lim_var, receiver_id,
     warn, err
   )
+  
+  # Check only if `arm_default_vals` is a character vector
+  # Reason: Arbitrary values allowed in case multiple studies are included with different arm values
+  CM$assert(
+    container = err,
+    cond = is.character(arm_default_vals) | is.null(arm_default_vals),
```

Comment:
Not really a bug, because the end result is `logical(1)` as it should. Still, using the shorter (element-wise) operator instead of the longer (logical) operator disguises the purpose of the condition: 
```suggestion
    cond = is.character(arm_default_vals) || is.null(arm_default_vals),
```
However, **take a look at the next comment before accepting this suggestion**!